docs/plus/pod.md

```diff
@@ -57,4 +93,267 @@ const chart = new k.Chart(app, 'Chart');
 const pod = new kplus.Pod(chart, 'Pod', {
   serviceAccount: kplus.ServiceAccount.fromServiceAccountName('aws'),
 });
-```
\ No newline at end of file
+```
+
+## Select a pod(s)
+
+Pods can also be selected by various mechanisms. These selections are often used in other
+cdk8s+ API's, such as [pod selection](./pod.md#pod-selection) during scheduling.
+
+### Select pods with labels
+
+Selects all pods that have the `app=store` label.
+
+```ts
+const pods = kplus.Pod.labeled(kplus.LabelQuery.is('app', 'store'));
+```
+
+### Select pods with labels in a particular namespace
+
+Pod selection can also be scoped to specific namespaces.
+This is done using the `.namespaced` method, which can accept any [namespace selector](./namespace.md#select-a-namespaces).
+
+For example, select all pods that have the `app=store` label in the `backoffice` namespace:
+
+```ts
+const pods = kplus.Pod.labeled(kplus.LabelQuery.is('app', 'store'))
+  .namespaced(kplus.Namespace.named('backoffice'));
+```
```

Comment:
I'm curious if switching the `Pod.select()`-based API with a chaining-based API was mostly an ergonomics choice, or if there's any other benefits we get? I remember some folks saying the chaining APIs are awkward in some JSII languages, though I don't want that to necessarily dictate the design here. 

I have a slight bias towards the non-chaining version, since I reckon it might be easier to extend with more options if necessary in the future - but I'm interested in hearing your perspective 🙂. I also noticed the chaining implementation imposes an order on the fields (you have to specify labels and then namespaces) - I wonder if that could become awkward? Maybe we could offer both styles? 
```ts
const pods = kplus.Pods.select({
  selectors: [kplus.LabelQuery.is('app', 'store')],
  namespace: kplus.Namespace.named('backoffice'),
});
// or
const pods = kplus.PodSelection.of({ ... })
```

Reply:
I gave some more though after our conversation and decided you're right. 

The type-safety suggested by my API complicates the ergonomics too much, especially when network policies are introduced, which has a different selection heuristic. I went with `Pods.select`, and we will add runtime checks where necessary